new/simple-manager-blueprint.yaml

```diff
@@ -328,6 +331,7 @@ node_templates:
     properties:
       ssh_user: { get_input: ssh_user }
       ssh_key_filename: { get_input: ssh_key_filename }
+      agent_local_key_path: { get_input: agent_private_key_path }
```

Comment:
Without this property after install manager we don't have private part of key for agent. I delete it  if it does not needed for simple install.